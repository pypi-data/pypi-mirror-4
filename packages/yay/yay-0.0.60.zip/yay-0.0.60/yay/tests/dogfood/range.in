upper: 3

foo: ${range(upper)}
bar: ${range(1, sum(upper, 1))}

