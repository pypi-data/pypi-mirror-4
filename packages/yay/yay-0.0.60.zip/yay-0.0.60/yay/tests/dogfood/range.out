upper: 3

foo:
 - 0
 - 1
 - 2

bar:
 - 1
 - 2
 - 3

